React/Views/RCTView.h

```diff
@@ -16,6 +16,30 @@
 extern const UIAccessibilityTraits SwitchAccessibilityTrait;
 #endif // TODO(macOS ISS#2323203)
 
+#if TARGET_OS_OSX
+typedef NS_ENUM(NSInteger, RCTCursor) {
```

Comment:
Can we put this into `RCTCursor.h` header along with the `RCTConvert` definition into a `RCTCursor.m` file, much like how `RCTResizeMode` is handled?